next/components/layouts/AccountPageLayout.tsx

```diff
@@ -85,6 +85,9 @@ const AccountPageLayout = ({
           title: 'account:account_section_applications.navigation',
           icon: <MySubmissionsIcon className="h-6 w-6" />,
           url: ROUTES.MY_APPLICATIONS,
+          // TODO this is a temporary solution until we have a proper way to handle this through authenticated page protection
```

Comment:
Add login checks like:
https://github.com/bratislava/konto.bratislava.sk/pull/661/commits/99ee6421091b05bac63b78416c643208448607fd

and remove this

Reply:
This can be removed then also:
https://github.com/bratislava/konto.bratislava.sk/blob/99ee6421091b05bac63b78416c643208448607fd/next/components/layouts/AccountPageLayout.tsx#L44